src/vm/excep.cpp

```diff
@@ -5665,6 +5665,8 @@ LONG __stdcall COMUnhandledExceptionFilter(     // EXCEPTION_CONTINUE_SEARCH or
         return retVal;
     }
 
+#ifndef FEATURE_CORECLR
+#ifdef FEATURE_CORRUPTING_EXCEPTIONS
```

Comment:
This is not correct. CSE related asserts are the ones that were under the "if" in your original change. This entire assert is not CSE specific.